src/main/java/justbot/storage/Storage.java

```diff
@@ -0,0 +1,177 @@
+package justbot.storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
```

Comment:
Maybe before this line, you could give a line break so that the ordering is more obvious!